impl/src/test/java/org/ehcache/impl/internal/statistics/DefaultCacheStatisticsTest.java

```diff
@@ -41,14 +56,39 @@
   private DefaultCacheStatistics cacheStatistics;
   private CacheManager cacheManager;
   private InternalCache<Long, String> cache;
-  private TestTimeSource timeSource = new TestTimeSource(System.currentTimeMillis());
+  private final TestTimeSource timeSource = new TestTimeSource(System.currentTimeMillis());
+  private final AtomicLong latency = new AtomicLong();
+  private final List<CacheEvent<? extends Long, ? extends String>> expirations = new LinkedList<>();
```

Comment:
Why a LinkedList?

Reply:
first version of test was using a queue. didn't change.